src/components/Integrations/Shopify/Shopify.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import Loading from '../../Loading/Loading';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const Shopify = ({ dependencies: { shopifyClient } }) => {
+  const [shops, setShops] = useState([]);
+  const [isConnected, setIsConnected] = useState(false);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const getData = async () => {
+      const result = await shopifyClient.getShopifyData();
+      if (result.success && result.value.length) {
+        setIsConnected(true);
+        setShops(result.value);
+      } else if (
+        !result.success &&
+        result.expectedError &&
+        result.expectedError.cannotConnectToAPI
+      ) {
+        setError('Error: No hemos podido conectar con la Api de Shopify, vuelve a intentar luego.');
+      } else if (!result.success) {
```

Comment:
maybe its better ask first this and then dont need to repeat ask for `!result.success`

Reply:
The alternative are _nested ifs_, and they tend to be less clear. Intendation is our enemy! :P

Reply:
maybe can use something like that too.
```js
if(result.expectedError &&
        result.expectedError.cannotConnectToAPI){}
    else if(!result.sucess){}
    else if(result.value.length){}
```js

Reply:
I will try this out. Thanks!

Reply:
I went out perfect! Thanks!